integration-test/1838-too-many-bays.py

```diff
@@ -33,9 +33,9 @@ def _bay(osm_id, area, name):
         )
 
         with self.features_in_tile_layer(z, x, y, 'water') as features:
-            # should only have top 10 features
-            self.assertEqual(10, len(features))
+            # should only have top 2 features
+            self.assertEqual(2, len(features))
```

Comment:
At zoom 8 our area threshold for labels is 500,000,000 (was 200,000,000) and for polygons it's still 20000000.

Under the earlier system 5 meet the area threshold, but after only 2 do. So the changes here make sense to me.